### PR TITLE
[IMP] website_crm_livechat : link visitor to lead on /lead command

### PR DESCRIPTION
**Pull request by dbeguin**
If an operator executes the /lead commands, the visitor was not linked to the visitor.
This commit fixed that. The visitor is now linked to the lead but the visitor does not
take the lead's name.
(As '/lead something' creates a lead with 'something' as name. 'Something is not the contact
name)

This PR also includes somes fixes : 
- Add acls to salesmen to see visitors and page views : The security file was not loaded in the manifest. Also, the tracking model name has changed to website_track.
- Allow only livechat users to see lead livechat sessions

Done in v13 and not in master as this functionality is needed to make crm visitor integration work properly.

Task ID: 2081534

### CODE REVIEW

File: addons/crm_livechat/models/mail_channel.py [hunk 1]
```diff
@@ -21,20 +21,24 @@ def _execute_command_lead(self, **kwargs):
...
+            'name': html2plaintext(key[5:]),
+            'partner_id': channel_partners.partner_id.id,
+            'user_id': None,
+            'team_id': None,
+            'description': html2plaintext(description),
+            'referred': partner.name,
+            'source_id': utm_source and utm_source.id,
+        })
+        lead._onchange_partner_id()
+        return lead
```
Comment (awa-odoo) [inline addons/crm_livechat/models/mail_channel.py]: I'm deeply disappointed

File: addons/website_crm_livechat/models/mail_channel.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class MailChannel(models.Model):
+    _inherit = 'mail.channel'
+
+    def _convert_visitor_to_lead(self, partner, channel_partners, key):
+        lead = super(MailChannel, self)._convert_visitor_to_lead(partner, channel_partners, key)
```
Comment (awa-odoo) [inline addons/website_crm_livechat/models/mail_channel.py]: Maybe a super tiny little small docstring to explain what you're doing?

File: addons/website_crm_livechat/views/website_crm_lead_views.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
         <field name="arch" type="xml">
             <xpath expr="//button[@name='action_redirect_to_page_views']" position="after">
                 <button name="action_redirect_to_livechat_sessions" type="object" class="oe_stat_button" icon="fa-comment"
-                        attrs="{'invisible': [('visitor_sessions_count', '=', 0)]}">
+                        attrs="{'invisible': [('visitor_sessions_count', '=', 0)]}" groups="im_livechat.im_livechat_group_user">
```
Comment (awa-odoo) [inline addons/website_crm_livechat/views/website_crm_lead_views.xml]: This seems slightly unrelated. Maybe move to another commit?

Comment (awa-odoo) [inline addons/website_crm_livechat/views/website_crm_lead_views.xml]: Should you also apply the group on the field itself?

Comment (dbeguin) [reply addons/website_crm_livechat/views/website_crm_lead_views.xml]: Ho ziziss right ! Thanks

**Comment by awa-odoo** [comment]
@dbeguin Side note: rephrase a bit the second commit.
Clearly split the "does not take the name of the visitor because blabla".

Also:
> If an operator executes the /lead commands, the visitor was not linked to the visitor.

oopsy :D

**Comment by dbeguin** [comment]
@awa-odoo : comments applied !

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff
